### PR TITLE
added carousel Testimonial

### PR DESCRIPTION
**Pull request by Magduri**
Screenshot -
screencapture-192-168-0-30-6006-2021-03-31-11_05_47
Mobile -
screencapture-192-168-0-30-6006-2021-03-31-11_06_49

### CODE REVIEW

**Comment by ralph-dev** [comment]
@Magduri Attach a Video so you can showcase the swipe / click interaction btw

File: src/Containers/CarouselTestimonial/CarouselTestimonial.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+     * Returns the JSX element of each carousel item based on its index
+     */
+    const getCarouselItem = (index: number) => (
+        <Section key={index}>
+            <TitleDiv key={carouselTitle}>{carouselTitle}</TitleDiv>
+            <ImageDiv key={carouselImage}>
+                <img src={carouselImage} />
+            </ImageDiv>
+            <ReviewTextDiv>
+                <SmallText color="#006aff" size="h4" bold>
```
Comment (ralph-dev) [inline src/Containers/CarouselTestimonial/CarouselTestimonial.tsx:36]: Use Theme Colors not Magic Coloors

**Review by ralph-dev — COMMENTED** [review]
Let's jump on a call to review please book a time

File: src/Containers/CarouselTestimonial/CarouselTestimonial.tsx (hunk 1 above)
Comment (Magduri) [reply src/Containers/CarouselTestimonial/CarouselTestimonial.tsx]: video:

https://user-images.githubusercontent.com/63363693/113187668-1e05ab80-9227-11eb-875d-8c449bf0e9bc.mp4

File: src/Containers/CarouselTestimonial/CarouselTestimonial.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+export default {
+    title: createStoryTitle('Carousel Testimonial'),
+    component: CarouselTestimonial,
+} as Meta;
+
+const defaultArgs = {
+    carouselTitle: 'Our customers love us on TrustPilot',
+    carouselImage:
+        'https://images.ctfassets.net/2d5q1td6cyxq/4eI7STmvwQ478Xhxnhwh4v/4c93a91efb4641907d2c8c2aafd67622/Trustpilot_Business__Collect_Customer_Service___Product_Reviews-6_1.svg',
+    reviewDescriptions: [
```
Comment (ralph-dev) [inline src/Containers/CarouselTestimonial/CarouselTestimonial.stories.tsx:18]: carouselData: [
{
  description:
  reviewer: 
}, {}
]

File: src/Containers/CarouselTestimonial/CarouselTestimonial.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+import { Carousel } from 'react-responsive-carousel';
+import styled from 'styled-components';
+import 'react-responsive-carousel/lib/styles/carousel.min.css';
+
+export interface ICarouselTestimonialProps {
+    carouselTitle: string;
+    carouselImage: string;
+    reviewDescriptions: string[];
+    reviewers: string[];
+    carouselAutoplay: boolean;
```
Comment (ralph-dev) [inline src/Containers/CarouselTestimonial/CarouselTestimonial.tsx:11]: is
or Has
isAutoPlaying
isLooping

File: src/Containers/CarouselTestimonial/CarouselTestimonial.tsx [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    );
+};
+
+const Section = styled.div`
+    margin: 10px 0px 10px 0px;
+`;
+const TitleDiv = styled.div`
+    font-size:16px;
+    color: black;
+    font-family: "Square Market", Helvetica, Arial, sans-serif;"
```
Comment (ralph-dev) [inline src/Containers/CarouselTestimonial/CarouselTestimonial.tsx:67]: Just use the CheaprEats

File: src/Containers/CarouselTestimonial/CarouselTestimonial.tsx [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+        </Carousel>
+    );
+};
+
+const Section = styled.div`
+    margin: 10px 0px 10px 0px;
+`;
+const TitleDiv = styled.div`
+    font-size:16px;
+    color: black;
```
Comment (ralph-dev) [inline src/Containers/CarouselTestimonial/CarouselTestimonial.tsx:66]: Colors from the Theme (CheaprEats)

File: src/Containers/CarouselTestimonial/CarouselTestimonial.tsx [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+    max-width: 90px;
+    margin-left: auto;
+    margin-right: auto;
+`;
+const ReviewTextDiv = styled.div`
+    margin-bottom: 14px;
+    margin-top: 28px;
+    font-size: 1.5em;
+    font-weight: bold;
+    color: #006aff ;
```
Comment (ralph-dev) [inline src/Containers/CarouselTestimonial/CarouselTestimonial.tsx:80]: Theme

**Review by ralph-dev — APPROVED** [review]
LGTM, Fix the comments, TEST it, Merge

**Pull request merged**